doc/protocols/scion-header.rst

```diff
@@ -787,19 +787,33 @@ Colibri Packet Timestamp
      0                   1                   2                   3
      0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1 2 3 4 5 6 7 8 9 0 1
     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-    |                             TsRel                             |
+    |                        Expiration Tick                        |
```

Comment:
Why did you move the Expiration Tick to the Colibri Packet Timestamp?
In my opinion, the Colibri Packet Timestamp should refer to the fields that help uniquely identify the packet, and which will be used by the duplicate suppression system - the Expiration Tick is not needed for that.
Also, the HVF is defined as
HVF_i = MAC_{\sigma_i}(PacketTimestamp, Original Packet Size)
It does not make sense to include the Expiration Tick here again as part of the packet timestamp, as it was already used in the computation of \sigma_i.

Reply:
Update: as discussed, the expiration tick is actually needed for the duplicate suppression, because TsRel is relative to the expiration tick.
Still, the computation of the HVF does not need to include the expiration tick.